chef/data_bags/crowbar/bc-template-updater.json

```diff
@@ -12,25 +12,32 @@
           "include_reboot_patches": false
         }
       },
-      "one_shot_run": false
+      "one_shot_run": false,
+      "upgrade_one_shot_run": false
     }
   },
   "deployment": {
     "updater": {
-      "crowbar-revision": 0,
+      "crowbar-revision": 1,
       "crowbar-applied": false,
       "elements": {},
       "element_states": {
-        "updater": [ "readying", "ready", "applying" ]
+        "updater": [ "readying", "ready", "applying" ],
+        "updater-crowbar-upgrade": [ "readying", "ready", "applying", "crowbar-upgrade" ]
       },
       "element_order": [
-        [ "updater" ]
+        [ "updater" ],
+        [ "updater-crowbar-upgrade" ]
       ],
+      "element_run_list_order": {
+        "updater": 40,
+        "updater-crowbar-upgrade": 2000
+      },
       "config": {
         "environment": "updater-config-base",
         "mode": "full",
-        "transitions": false,
-        "transition_list": []
+        "transitions": true,
+        "transition_list": [ "crowbar-upgrade" ]
```

Comment:
Unless we need to implement a ```transition``` method in  updater_service.rb I think we any changes in this section.